ddtrace/__init__.py

```diff
@@ -5,6 +5,11 @@
 
 LOADED_MODULES = frozenset(sys.modules.keys())
 
+# Configuration for the whole tracer from file. Do it before anything else happens.
+from ddtrace.internal.native import _apply_configuration_from_disk
+
+_apply_configuration_from_disk()
```

Comment:
Just a random thought on whether it is ok to continue doing stuff with side-effects on module import. Down the line we might find these as blockers towards "pure" modules

Reply:
Yeah, the `_apply_configuration_from_disk` should definitely be removed down the line. We should centralize config application and do it here; but that's quite some work and will be done in other PRs.